totest-manager.py

```diff
@@ -222,6 +230,56 @@ def update_openqa_status_message(self):
                 self.openqa.openqa_request(
                     'POST', 'groups/%s/comments' % group_id, data=data)
 
+    def _get_repo_mirror(self, snapshot):
+        """Get the URL of the repo mirror from openQA"""
+        jobs = self.find_openqa_results(snapshot)
+        # Find the MIRROR_HTTP variable, most jobs have it
+        http_mirror = None
+        for job in jobs:
+            if 'settings' in job and 'MIRROR_HTTP' in job['settings']:
+                return job['settings']['SUSEMIRROR']
+
+    def _download_file(self, url):
+        try:
+            req = urllib2.Request(url=url)
+            f = urllib2.urlopen(req)
+            return f.read()
+        except urllib2.HTTPError:
+            return None
+
+    def _get_repo_primary(self, baseurl):
```

Comment:
We now have code like this in at least 2 places, presumably this was copied or based on one. Seems like a candidate for `osclib.util`.

Reply:
I'll use `requests` instead, that's a one line call as well.